astropy/io/ascii/html.py

```diff
@@ -265,6 +264,18 @@ class HTML(core.BaseReader):
             use the syntax 1.36583e-13 .. 1.36583e-13 for output. If not
             present, this parameter will be true by default.
 
+        * raw_html_cols : column name or list of names with raw HTML content
+            This allows one to include raw HTML strings in the column output,
+            for instance to include link references in a table.  Normally
+            the HTML characters are escaped
+
+        * raw_html_clean_kwargs : dict of keyword args controlling HTML cleaning
+            If the ``bleach`` package is installed then any raw HTML will be
+            cleaned to prevent unsafe HTML from ending up in the table output.
+            This is done by calling ``bleach.clean(data, **raw_html_clean_kwargs)``.
+            For details on the available options (e.g. tag whitelist) see:
+            http://bleach.readthedocs.org/en/latest/clean.html
```

Comment:
I feel inclined to make this a requirement to use the feature at all.  If it sweetens the deal though we could include `bleach` in `install_requires` so that it will almost always be installed along with Astropy anyways (I see no reason not to add more install dependencies these days, especially if they're pure Python....)